coepelduynen_main_iterator.py

```diff
@@ -40,7 +40,7 @@
                 parts=4,
                 normalize_scaler=loaded_model["scaler"],
                 aggregate=False,
-                resolution_aggregate=0.3,
+                resolution_aggregate=0.2,
```

Comment:
Is this logical? This is smaller than the actual resolution of the images I believe?